### PR TITLE
change active role color to forest

### PR DESCRIPTION
**Pull request by omnerom**
role description is already in forest which is weird that it doesn't match

### CODE REVIEW

**Comment by Jurorno9** [comment]
mmm, the code is not compiled.

Im sure it's fine

**Comment by omnerom** [comment]
👍

File: src/ranks.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export class Rank {
 		blocksPlaced: 5000,
 		timeSinceFirstJoin: 24 * 60 * 60 * 1000 * 7, //7 days
 	});
-	static trusted = new Rank("trusted", 2, "Trusted players who have gained the trust of a mod or admin.", "[black]<[#E67E22]\uE813[]>[]", "&y[T]&fr", "[#E67E22]");
+	static trusted = new Rank("trusted", 2, "Trusted players who have gained the trust of a mod or admin.", "[black]<[forest]\uE813[]>[]", "&y[T]&fr", "[#E67E22]");
```
Comment (BalaM314) [inline src/ranks.ts:20]: I think you meant to change it for active rank, not trusted rank, right?

**Review by BalaM314 — CHANGES_REQUESTED** [review]

File: src/ranks.ts (hunk 1 above)
Comment (omnerom) [reply src/ranks.ts:20]: What the hail, yeah. Must have been blind when I did that

Comment (BalaM314) [reply src/ranks.ts:20]: @omnerom you can make another edit in the same PR, or creating another one is fine too

Comment (omnerom) [reply src/ranks.ts]: Edited both ranks.js and ranks.ts correctly

File: src/ranks.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ export class Rank {
 	static autoRanks: Rank[] = [];
 
 	static player = new Rank("player", 0, "Ordinary players.", "", "&lk[p]&fr", "");
-	static active = new Rank("active", 1, "Assigned automatically to players who have played for some time.", "[black]<[#E67E22]\uE800[]>[]", "&g[P]&fr", "[forest]", {
+	static active = new Rank("active", 1, "Assigned automatically to players who have played for some time.", "[black]<[#forest]\uE800[]>[]", "&g[P]&fr", "[forest]", {
```
Comment (BalaM314) [inline src/ranks.ts:14]: This is saying `[#forest]`, which won't work, it needs to be `[forest]`

**Review by BalaM314 — CHANGES_REQUESTED** [review]

File: src/ranks.ts (hunk 2 above)
Comment (omnerom) [reply src/ranks.ts]: Fixed

**Review by BalaM314 — APPROVED** [review]
Looks good!

**Comment by BalaM314** [comment]
Congratulations on your very first PR!

**Pull request merged**